components/topics/topic-adder.jsx

```diff
@@ -0,0 +1,23 @@
+
+const AddTopic = ()=>{
```

Comment:
Is this a part of this ticket? Try to keep the code you submit to the ticket at hand.